src/noisepy/seis/S2_stacking.py

```diff
@@ -221,8 +221,8 @@ def stack(rootpath: str, stack_method: str):
             except Exception:
                 if flag:
                     print("continue! no pair of %s in %s" % (dtype, ifile))
-                continue
-
+                continue          
+            print(path_list)
```

Comment:
@mdenolle Can you remove this print or change it  to use the `logger` if its needed?